thrift-parser.js

```diff
@@ -440,7 +440,7 @@ module.exports = (buffer, offset = 0) => {
     let subject = readKeyword('include');
     readSpace();
     let includePath = readQuotation();
-    let name = includePath.replace(/^.*?([^/]*?)(:?\.thrift)?$/, '$1.thrift');
+    let name = includePath.replace(/^.*?([^/\\]*?)(:?\.thrift)?$/, '$1');
```

Comment:
Why is there an explicit `:` reference?  The IDL says it can be any character between the quotations (https://thrift.apache.org/docs/idl#literal).

I'm viewing the regexp with: https://regexper.com/#%2F%5E.*%3F(%5B%5E%2F%5C%5C%5D*%3F)(%3A%3F%5C.thrift)%3F%24%2F